### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by jama79**
In this project, an HTML, CSS, and javascript document was started. Later a new Branch was created to which Linterns were added for each of the documents created in the Main Branch

### CODE REVIEW

File: Hello-world.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-const == 10;
+const number = 3;
 let word = "Hello World";
-console.lof(word);
\ No newline at end of file
+console.log(word);
```
Comment (enaburekhan) [inline Hello-world.js]: - [ ] I will encourage you to follow the project requirements by removing the `Hello-world.js` file. We really don't need it now

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (enaburekhan) [inline .gitignore]: - [ ] It will be nice to add the node-modules file to the .gitignore. As it can be added by accident while setting up linters

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi Jama79, 

Good job so far!

- [ ] Project highlight

- The codebase is clean, easy to read and understand.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] I observed there was no readme added to this project. Kindly use this readme [template](https://github.com/microverseinc/readme-template) to add a descriptive readme to your project.
- [ ] I will also suggest you add linters.yml config file to your project using this [link](https://github.com/microverseinc/linters-config/tree/master/html-css). Read the `set-up github actions` and follow the steps. Currently, your PR is not linting because this file is missing.

- [ ] It will be nice to have a title for your pull request (PR). And the PR commit messages should be more descriptive. Example:

```
#Bad commit messages
gitignore
add .eslint

# Good commit messages
Add gitignore
Add .eslint
```
check this [page](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) on pathwright for more infomation.

- [ ] I will also suggest you make your branch name to be more descriptive. Instead of `feature`, you can use a more meaningful name like `hello-world`

 __Check the inline code-reviews below__

`Gentle reminder`: I had to write most of the comments here instead of in the codebase because the files/codes to comment on are not present 

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: .gitignore (hunk 2 above)
Comment (jama79) [reply .gitignore]: node_modules was added

**Pull request closed without merging**